src/@newrelic/gatsby-theme-newrelic/components/Logo.js

```diff
@@ -41,10 +41,15 @@ const Logo = ({ className, width }) => {
     <IOLogo className={className} width={width} />
   );
 
-  return window.location &&
-    window.location.href.includes('instant-observability')
-    ? instantObservabilityLogo
-    : developerLogo;
+  const [logo, setLogo] = useState(developerLogo);
+
+  useEffect(() => {
+    if (window.location.pathname.includes('instant-observability')) {
```

Comment:
We _might_ also want to check if `window` exists so that this doesn't yell at us at build time.

Reply:
i believe window is always available in useEffect because it only runs after the render and not server side, but we could put a check in there for extra safety!